tests/test_adresse.py

```diff
@@ -21,14 +24,72 @@ def test_serialization_only_required_fields(self):
             hausnummer=address_test_data["hausnummer"],
         )
 
-        address_json = a.dumps(
-            strip_nulls=True,
-            key_transformer=jsons.KEY_TRANSFORMER_CAMELCASE,
-            jdkwargs={"ensure_ascii": False},
-        )
+        schema = AdresseSchema()
+        address_json = schema.dumps(a, ensure_ascii=False)
 
+        assert "Nördliche Münchner Straße" in address_json
+        assert "27A" in address_json
```

Comment:
`in` assertions auf eigentlich strukturierte daten finde ich nicht soo schön. ist aber nicht wichtig.
Besser fände ich:
```python
json_dict = json.loads(address_json)
assert json_dict["strasse"] == "Nördliche Münchner Straße"
```

Aber, wie gesagt, niicht wichtig.

Reply:
Kann ich gerne anpassen. Mein Gedanke war nur, dass für das Backend eigentlich immer nur strings hin und her geschickt werden. Sehe aber den Punkt dass deine Annahme präziser ist.
Denke wir können die Annahmen an deinen Vorschlag anpassen und uns darauf verlassen, dass marshmallow die Umwandlung eines dictionaries in einen fromatierten string schon hinbekommt.

Reply:
Habe die Tests nun umstrukturiert zu Dictionaries anstatt strings.